### ISSUE TITLE
Not able to reproduce same CoLA result as huggingface defualt

### ISSUE THREAD

**Issue by leo-liuzy** [huggingface/transformers#4086]
# 🐛 Bug
When I run your official `run_glue.py` script on CoLA, the performance is significantly lower than the ones in your website. Mine: ~10, Yours: ~50
## Information

Model I am using (Bert, XLNet ...):
bert-base-uncased
Language I am using the model on (English, Chinese ...):
English

The problem arises when using:
* [x] the official example scripts: (give details below)
* [ ] my own modified scripts: (give details below)

The tasks I am working on is:
* [x] an official GLUE/SQUaD task: (give the name)
* [ ] my own task or dataset: (give details below)

## To reproduce

1. Run my provided command line
2.
3.
```
export GLUE_DIR=downstream_datasets/glue
export TASK_NAME=CoLA
SEED=43
export CUDA_VISIBLE_DEVICES=0

python run_glue.py \
  --model_type bert \
  --model_name_or_path bert-base-uncased \
  --task_name $TASK_NAME \
  --do_train \
  --save_steps 200 \
  --do_eval \
  --data_dir $GLUE_DIR/$TASK_NAME \
  --max_seq_length 128 \
  --per_gpu_train_batch_size 32 \
  --learning_rate 2e-5 \
  --num_train_epochs 30 \
  --seed $SEED \
  --output_dir tmp/"$TASK_NAME"_seed"$SEED"/
```

## Environment info

     
- `transformers` version: 1.17.2
- Platform: CentOS 7.7 64-bit
- Python version: python 3.7
- PyTorch version (GPU?): 1.5.0+cuda9.2
- Tensorflow version (GPU?):
- Using GPU in script?: Yes
- Using distributed or parallel set-up in script?: No, single machine

**Comment by julien-c** [comment]
Do you have a link to a TensorBoard or any other experiment tracking for your training?

You can also add the `--evaluate_during_training` flag to see the evolution of the eval metric during training.

**Comment by ivokun** [comment]
Have you tried with fewer epochs? (e.g ```--num_train_epochs 3.0``` as stated in the repo's example)

Also what you need to consider is seed hyper-parameter.
